app/helpers/logging_helper.rb

```diff
@@ -0,0 +1,34 @@
+module LoggingHelper
```

Comment:
The thread local data should be cleared after the HTTP request has been completed. This should be done in an `ensure` block so that no matter what errors are raised the data is cleared.